index.js

```diff
@@ -16,39 +16,33 @@ class HyperCoreLoggerFacility extends Base {
   }
 
   _start0 (cb) {
-    super._start0(err => {
```

Comment:
because none other implementation do that and that super has no logic in it, it's just what you pass in there

Reply:
the only purpose of `_start0` is to be called first, before anything else gets initiated/called as far as I can see

Reply:
Usually it's a good practice to call the parent's function, unless you spefically know you don't want it to run.
In the future, when `_start0` has some logic in it, I don't expect the author to assume it's not going to be called.

Since it's a no-op, I think we shall keep it in.
Not a big deal though. Approving.